go.mod

```
@@ -23,40 +23,43 @@ require (
 	go.uber.org/zap v1.13.0 // indirect
 	golang.org/x/net v0.0.0-20210525063256-abc453219eb5
 	golang.org/x/sys v0.0.0-20210903071746-97244b99971b
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
-	google.golang.org/grpc v1.31.0
-	k8s.io/api v0.19.4
-	k8s.io/apimachinery v0.22.1
-	k8s.io/client-go v0.19.4
-	k8s.io/cri-api v0.0.0
-	k8s.io/klog/v2 v2.9.0
-	k8s.io/kubernetes v1.19.4
+	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
+	google.golang.org/grpc v1.33.2
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
+	k8s.io/cri-api v0.21.0
+	k8s.io/klog/v2 v2.8.0
+	k8s.io/kubernetes v1.21.0
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
 	github.com/intel/cri-resource-manager/pkg/topology v0.0.0 => ./pkg/topology
 	google.golang.org/grpc => google.golang.org/grpc v1.26.0
 
-	k8s.io/api v0.0.0 => k8s.io/api v0.0.0-20201113170447-7ec4e34ebfa0
-	k8s.io/apiextensions-apiserver v0.0.0 => k8s.io/apiextensions-apiserver v0.0.0-20201113174919-8ce2dfad2388
-	k8s.io/apimachinery v0.0.0 => k8s.io/apimachinery v0.0.0-20200821171749-b63a0c883fbf
-	k8s.io/apiserver v0.0.0 => k8s.io/apiserver v0.0.0-20201113172959-d4704a3a5cde
-	k8s.io/cli-runtime v0.0.0 => k8s.io/cli-runtime v0.0.0-20201113175539-7ac36a2d8758
-	k8s.io/client-go v0.0.0 => k8s.io/client-go v0.0.0-20201113171635-2bb8681d6833
-	k8s.io/cloud-provider v0.0.0 => k8s.io/cloud-provider v0.0.0-20201113181133-070bf588610e
-	k8s.io/cluster-bootstrap v0.0.0 => k8s.io/cluster-bootstrap v0.0.0-20201113181704-acbca43bf834
-	k8s.io/code-generator v0.0.0 => k8s.io/code-generator v0.0.0-20200813171329-1c3794b15e35
-	k8s.io/component-base v0.0.0 => k8s.io/component-base v0.0.0-20201113171950-93d3585cb60c
-	k8s.io/cri-api v0.0.0 => k8s.io/cri-api v0.0.0-20200727095024-0fd2b9b35a65
-	k8s.io/csi-translation-lib v0.0.0 => k8s.io/csi-translation-lib v0.0.0-20201113182001-d03728bbcf5a
-	k8s.io/kube-aggregator v0.0.0 => k8s.io/kube-aggregator v0.0.0-20201113173433-ce05bf9f9509
-	k8s.io/kube-controller-manager v0.0.0 => k8s.io/kube-controller-manager v0.0.0-20201113181424-f91ce16d8de8
-	k8s.io/kube-proxy v0.0.0 => k8s.io/kube-proxy v0.0.0-20201113180133-d8e6aa5a8916
-	k8s.io/kube-scheduler v0.0.0 => k8s.io/kube-scheduler v0.0.0-20201113180716-8fee79fb9051
-	k8s.io/kubectl v0.0.0 => k8s.io/kubectl v0.0.0-20201113183129-6c3805304969
-	k8s.io/kubelet v0.0.0 => k8s.io/kubelet v0.0.0-20201113180421-609649667e1d
-	k8s.io/legacy-cloud-providers v0.0.0 => k8s.io/legacy-cloud-providers v0.0.0-20201113182353-b77454164b5d
-	k8s.io/metrics v0.0.0 => k8s.io/metrics v0.0.0-20201113175240-4a8e322202a4
-	k8s.io/sample-apiserver v0.0.0 => k8s.io/sample-apiserver v0.0.0-20201113174004-1f754d3b7d52
+	k8s.io/api v0.0.0 => k8s.io/api v0.0.0-20210408192244-5d0d5b56b511
```

```suggestion
	k8s.io/api => k8s.io/api v0.21.0
```

and ditto X times below...